src/components/CardSchemes/components/PaymentMethodIcon/PaymentMethodIcon.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { keys, values } from 'lodash/fp';
+import PaymentMethodIcon from './PaymentMethodIcon';
+// eslint-disable-next-line max-len
+import schemeMap, {
+  iconComponents
+} from '../../../CreditCardDetails/components/scheme-icons/card-scheme-icons';
+
+describe('PaymentMethodIcon', () => {
+  /**
+   * Style tests.
+   */
+  it('should render with default styles', () => {
+    const actual = create(
+      <div>
+        {keys(schemeMap).map(schemeId => (
+          <PaymentMethodIcon
+            size={PaymentMethodIcon.KILO}
+            key={schemeId}
+            schemeId={schemeId}
+          />
+        ))}
+      </div>
+    );
+    expect(actual).toMatchSnapshot();
+  });
+
+  it('should render svg icon', () => {
```

Comment:
This is basically the test I was asking for under the `CardSchemes` component. To check explicitly that each icon in the `schemeIds` prop gets rendered.

I would love to see a slightly more descriptive description. Something like

> should render each icon specified in the "schemeIds" prop